### PR TITLE
Run bash and daisy in one command

### PR DESCRIPTION
**Pull request by koln67**
Set the secret variable before running daisy, all in one bash command

### CODE REVIEW

File: prow/prowjobs/GoogleCloudPlatform/gcp-guest/compute-image-tools.yaml [hunk 1]
```diff
@@ -15,11 +15,7 @@ presubmits:
         - "bash"
         args:
         - "-c"
-        - "sbom_root=$(gcloud secrets versions access latest --secret=sbom-util-secret --project=gcp-guest); "
-        - "/daisy"
-        - "project=compute-image-test-pool-001" 
-        - "var:sbom_util_gcs_root=$sbom_root"
-        - "daisy_workflows/sbom_validation/enterprise_sbom_test.wf.json"
+        - "sbom_root=$(gcloud secrets versions access latest --secret=sbom-util-secret --project=gcp-guest); /daisy -project=studious-sign-357617 -var:sbom_util_gcs_root=$sbom_root daisy_workflows/sbom_validation/enterprise_sbom_test.wf.json"
```
Comment (Quintonamore) [inline prow/prowjobs/GoogleCloudPlatform/gcp-guest/compute-image-tools.yaml:18]: I've never heard of project `studious-sign-357617` why are we using that?

Comment (koln67) [reply prow/prowjobs/GoogleCloudPlatform/gcp-guest/compute-image-tools.yaml]: fixed, that was what I was using when testing this manually but it should be compute-image-test-pool-001

**Pull request merged**